test/Libraries/RevitNodesTests/Elements/RoomTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Linq;
+
+using Autodesk.DesignScript.Geometry;
+
+using Revit.Elements;
+using NUnit.Framework;
+
+using RevitTestServices;
+
+using RTF.Framework;
+
+namespace RevitNodesTests.Elements
+{
+    [TestFixture]
+    public class RoomTests : RevitNodeTestBase
+    {
+        /// <summary>
+        /// Testing to create a room by location and query its name
+        /// </summary>
+        [Test]
+        [TestModel(@".\emptyAnnotativeView.rvt")]
+        public void Create_ValidArgs()
+        {
+            Level lvl = Level.ByElevation(0);
+            Assert.IsNotNull(lvl);
+
+            var room = Revit.Elements.Room.ByLocation(lvl,Point.ByCoordinates(0, 0, 0),"myRoom", "myNumber");
+            
+            Assert.NotNull(room);
+            Assert.AreEqual(room.InternalRevitElement.Name, "myRoom");      
+        }
+
```

Comment:
Also, it would be good to have tests to validate the update and element binding. Create a DYN file for room creation, run it and then modify some of the input parameters run it again. Validate that the hosting is maintained, the same room element is updated etc.